.devops/pagopa-code-review-pipelines.yml

```
@@ -34,7 +34,7 @@ resources:
     - repository: checkout-be-mock
       type: github
       name: pagopa/pagopa-checkout-be-mock
-      ref: master
+      ref: CHK-2779-redirect-outcome-mapping
```

```suggestion
      ref: master
```
TODO: to be replaced after dependent PR merge